tracks/csharp/exercises/sum-of-multiples/README.md

```diff
@@ -0,0 +1,60 @@
+### Reasonable solutions
+
+```csharp
+using System.Linq;
+using System.Collections.Generic;
+
+public static class SumOfMultiples
+{
+    public static int Sum(IEnumerable<int> inputs, int max)
```

Comment:
Could you rename `inputs` to `multiples` (which is the parameter's name in the [latest version of the exercise](https://github.com/exercism/csharp/blob/master/exercises/sum-of-multiples/SumOfMultiples.cs))?

Reply:
I think multiples is a misnomer but have adopted it in the interests of consistent house standards